linux_os/guide/system/accounts/accounts-restrictions/account_expiration/ensure_shadow_group_empty/tests/shadow_user_member.fail.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+# platform = multi_platform_sle,multi_platform_ubuntu
+# remediation = none
```

Comment:
I believe it is fine to remediate this test.

Reply:
it will fail, we don't touch the `/etc/passwd` file in the remediation

Reply:
Got it. Thanks.